### PR TITLE
rework the site for 1.0.0

### PR DESCRIPTION
**Pull request by rkuhn**
- removed footer since the list of projects speaks for itself
- added working groups
- corrected links to now-moved repositories
- link to the published API docs instead of the github repo
- polish the CSS a little

### CODE REVIEW

**Comment by rkuhn** [comment]
This encompasses #11 (@purplefox please verify).

@reactive-streams/contributors please check this for correctness. We need to push out an update to correct the now-dead links to the moved repositories in any case.

@reactive-streams/js-admin please take a look at the summary of the JavaScript working group, I hope I got the gist correctly, correct me if I’m wrong.

File: _includes/index.md [hunk 1]
```diff
@@ -1,60 +1,85 @@
...
+This working group defines network protocols for passing reactive streams over various transport media that involve serialization and deserialization of the data elements. Examples of such transports are TCP, UDP, HTTP and WebSockets.
 
-### Implementations of the draft spec
+This work is performed in the [reactive-streams-io](https://github.com/reactive-streams/reactive-streams-io/) repository.
 
-* Akka Streams
-   * See this [Activator template](http://www.typesafe.com/activator/template/akka-stream-scala) introducing the [Akka Project](http://akka.io/) implementation in Scala; a Java version will follow shortly.
+## Current State
+
+As of Feb 16, 2015 we have released version 1.0.0-RC3 of Reactive Streams for the JVM, including Java [API](http://www.reactive-streams.org/reactive-streams-1.0.0.RC3-javadoc), a textual [Specification](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.0.RC3/README.md#specification) and a [TCK](http://www.reactive-streams.org/reactive-streams-tck-1.0.0.RC3-javadoc). Corresponding code artifacts are available on Maven Central:
```
Comment (viktorklang) [inline _includes/index.md]: Update this section to reflect 1.0.0.RC4 on April 1st (no joke) ;)

File: _includes/index.md [hunk 2]
```diff
@@ -1,60 +1,85 @@
...
+      <version>1.0.0.RC3</version>
+    </dependency>
+
+The source code for these is available on [github](https://github.com/reactive-streams/reactive-streams-jvm/tree/v1.0.0.RC3). Please use github issues for providing feedback.
+
+All artifacts and specifications are released under [Creative Commons Zero](http://creativecommons.org/publicdomain/zero/1.0) into the Public Domain.
+
+### Implementations (sorted alphabetically)
+
+#### On the JVM
```
Comment (viktorklang) [inline _includes/index.md]: I think this section should also list what version of the spec the implementations conform to. Wdyt?

File: _includes/index.md [hunk 3]
```diff
@@ -1,60 +1,85 @@
...
+To get started implementing the draft specification, it is recommended to start by reading the [README](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.0.RC3/README.md), then taking a look at the [Specification](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.0.RC3/README.md#specification) then taking a look at the [TCK](https://github.com/reactive-streams/reactive-streams-jvm/tree/v1.0.0.RC3/tck). If you have an issue with any of the above, please take a look at [closed issues](https://github.com/reactive-streams/reactive-streams/issues?page=1&state=closed) and then open a [new issue](https://github.com/reactive-streams/reactive-streams/issues/new) if it has not already been answered.
```
Comment (viktorklang) [inline _includes/index.md]: Add the javadoc links?

**Comment by viktorklang** [comment]
Reviewed, :+1: aside from the comments.
given that people have had a month to review this, I think it is safe to merge.

**Comment by smaldini** [comment]
Is there still time to edit a small change regarding RC3 to RC4, and also a quick amend to Reactor (it's actually supported in Reactor from 2.0 not 1.1)

**Comment by smaldini** [comment]
Actually I can do that post merge since they are really small changes.

**Comment by viktorklang** [comment]
Oh, for sure!

**Comment by smaldini** [comment]
Actually @rkuhn did already update the link! Good to merge in essence, if there is something else I'll do another PR.

File: _includes/index.md (hunk 1 above)
Comment (rkuhn) [inline _includes/index.md]: will do

File: _includes/index.md (hunk 2 above)
Comment (rkuhn) [inline _includes/index.md]: yes, good point; right now we have Reactor, RxJava, vert.x and Akka on RC4, right?

**Comment by rkuhn** [comment]
I’ll make the Reactor update as part of the other fixes, should be quick.

File: _includes/index.md (hunk 2 above)
Comment (viktorklang) [inline _includes/index.md]: Yes, AFAIK

**Comment by rkuhn** [comment]
I merged my changes with the master branch, now all good from my side.

**Comment by viktorklang** [comment]
I'm OK with merging.

**Comment by rkuhn** [comment]
@smaldini It seems that the 2.0 vs 1.1 concern was implicitly addressed already, as I couldn’t find that on the site. If something is still amiss then it can be fixed easily.

**Comment by benjchristensen** [comment]
:+1:

File: _includes/index.md [hunk 4]
```diff
@@ -1,60 +1,85 @@
...
    * See the [“Streams”](http://www.ratpack.io/manual/current/streams.html) chapter of the manual.
+* Reactive Rabbit *(tested with TCK 1.0.0.RC3)*
+   * Driver for RabbitMQ/AMQP.
+   * See [github.com/ScalaConsultants/reactive-rabbit](https://github.com/ScalaConsultants/reactive-rabbit).
+* [Reactor](http://projectreactor.io/) *(tested with TCK 1.0.0.RC4)*
+   * For the documentation see [here](http://projectreactor.io/docs/reference/streams.html).
+* [RxJava](http://reactivex.io/) *(tested with TCK 1.0.0.RC4)*
+   * See [github.com/ReactiveX/RxJavaReactiveStreams](https://github.com/ReactiveX/RxJavaReactiveStreams).
+* [Vert.x 3.0](http://vertx.io) *(tested with TCK 1.0.0.RC4)*
+   * Vert.x 3.0 is currently in alpha. The reactive streams implementation can be found [here](https://github.com/vert-x3/ext/tree/master/ext-reactive-streams).
```
Comment (benjchristensen) [inline _includes/index.md]: This URL does not work.

Comment (viktorklang) [inline _includes/index.md]: Nice catch, will change to https://github.com/vert-x3/vertx-reactive-streams

Comment (viktorklang) [inline _includes/index.md]: https://github.com/reactive-streams/reactive-streams.github.io/commit/718a1a7d4ff153031080d64c52195e5357fc8c54

**Pull request merged**